src/libraries/Microsoft.PowerFx.Connectors/OpenApiExtensions.cs

```diff
@@ -271,8 +271,7 @@ public static ConnectorParameterType ToFormulaType(this OpenApiSchema schema, St
 
                     switch (schema.Format)
                     {                        
-                        case "date":
-                            return new ConnectorParameterType(schema, FormulaType.Date);
```

Comment:
>Date [](http://example.com/codeflow?start=82&length=4)

fwiw - This change surprises me.  
Are we confident it's safe?

Reply:
This "date" format is uncommon and wasn't tested